src/app/features/tech-record/components/tech-record-amend-vrm/tech-record-amend-vrm.component.html

```diff
@@ -0,0 +1,44 @@
+<div *ngIf="vehicle">
+  <h2 class="govuk-heading-l">{{ makeAndModel }}</h2>
+
+  <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Vehicle type</dt>
+      <dd id="vehicleType" class="govuk-summary-list__value">
+        {{ currentTechRecord?.vehicleType | defaultNullOrEmpty | uppercase }}
+      </dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">VRM</dt>
+      <app-number-plate *ngIf="vrm" [vrm]="vrm"></app-number-plate>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">VIN/chassis number</dt>
+      <dd id="vehicleType" class="govuk-summary-list__value">{{ vehicle.vin | defaultNullOrEmpty }}</dd>
+    </div>
+  </dl>
+
+  <form [formGroup]="form">
+    <app-text-input
+      #newVrm
+      [label]="form.meta.children![0].label"
+      [formControlName]="form.meta.children![0].name"
+      [width]="width"
+      name="amend-vrm-text"
+      (ngModelChange)="ngOnChanges(); newVrm.value = newVrm.value.toUpperCase()"
+    ></app-text-input>
+
+    <div *ngIf="newVrm.value.toUpperCase() !== vrm && newVrm.value.length">
+      <p class="govuk-heading-l">Are you sure you want to change this VRM?</p>
+      <p class="govuk-body">Amending this VRM will also amend the VRM for all other tech records present.</p>
+    </div>
+
+    <div class="buttons-row">
```

Comment:
Could be using app-button-group and app-button

Reply:
Wrapping it just in an app-button-group breaks the vertical alignment of the buttons, is there something else I'd need to add to the group to fix that?

Reply:
`<app-button-group>`
      `<app-button id="submit-vrm" (clicked)="handleSubmit(newVrm.value)">Save</app-button>`
      `<app-button design="link" data-module="govuk-button" (clicked)="navigateBack()">Back</app-button>`
`</app-button-group>`
Does something like this not work?

Reply:
Yeah, what you've put works, what I tried didn't